docs/modules/components/form/input/ts/views/result-input-date.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
```

Comment:
Change the react import removing the *. @Gabojcb

Reply:
Now i do